langroid/agent/special/doc_chat_agent.py

```diff
@@ -1315,6 +1388,35 @@ def get_relevant_extracts(self, query: str) -> Tuple[str, List[Document]]:
 
         return query, extracts
 
+    def clean_generated_content(self, passages: List[Document]) -> List[Document]:
```

Comment:
suggest rename - `remove_hypothetical_questions` to be more clear

Reply:
I thoughy it might need for other cleaning stuff so I used a generic name, but ok I'll make it specific for this task